Dockerfile

```diff
@@ -1,86 +1,74 @@
 # syntax = docker/dockerfile:1
 
-# Make sure RUBY_VERSION matches the Ruby version in .tool-versions
+# Define Ruby version
 ARG RUBY_VERSION=3.3
 FROM public.ecr.aws/docker/library/ruby:$RUBY_VERSION-alpine AS base
 
-# Rails app lives here
+# Set working directory
 WORKDIR /rails
 
-# Set production environment
+# Configure environment variables
 ENV RAILS_ENV="production" \
     BUNDLE_DEPLOYMENT="1" \
     BUNDLE_PATH="/usr/local/bundle" \
     BUNDLE_WITHOUT="development test" \
-    BUNDLE_FORCE_RUBY_PLATFORM="1"
+    BUNDLE_FORCE_RUBY_PLATFORM="1" \
+    RUBY_YJIT_ENABLE="true"
 
+# Stage to download the gRPC health probe
 FROM public.ecr.aws/docker/library/alpine:latest AS binaries
 
-# Install packages needed to build binaries
-RUN apk update --no-cache && \
-    apk upgrade --no-cache && \
-    apk add --update --no-cache wget curl jq
-
-RUN [[ "$(arch)" = "aarch64" ]] && arch="arm64" || arch="$(arch)" && \
-  curl -s https://api.github.com/repos/grpc-ecosystem/grpc-health-probe/releases/latest \
-  | jq ".assets[].browser_download_url" \
-  | grep "grpc_health_probe-linux-$arch" \
-  | tr -d '"' \
-  | wget -qi - -O grpc-health-probe && \
-  chmod +x grpc-health-probe && \
-  mv grpc-health-probe /usr/local/bin
-
-# Throw-away build stage to reduce size of final image
+# Install packages to download the gRPC health probe
+RUN apk add --no-cache wget curl jq
+
+# Download and configure the gRPC health probe
+RUN ARCH=$(uname -m) && \
```

Comment:
You changed `arch` to `uname -m`. I can't remember why I used `arch` but I think it does the same as `uname -m` Also small thing but the static analysis here: https://app.codacy.com/gh/somleng/somleng/pull-requests/1721/issues recommends to quote the output. I know here it's redundant because that command shouldn't output newlines anyway.

Reply:
This item may be rolled back.